scripts/include.am

```diff
@@ -126,3 +126,8 @@ endif
 EXTRA_DIST += scripts/bench/bench_functions.sh
 
 EXTRA_DIST += scripts/user_settings_asm.sh
+
+EXTRA_DIST += \
+        scripts/config_check/README.md  \
```

Comment:
`help.txt` is missing. If this is just output from `./configure --help` then you should just document how to use that and not make a copy of build options that needs updated.

Reply:
I've removed the `help.txt` file and added a comment in the README.